master/docs/manual/cfg-reporters.rst

```diff
@@ -418,6 +422,51 @@ The default ``ctx`` for the missing worker email is made of:
 
 .. _Jinja2: http://jinja.pocoo.org/docs/dev/templates/
 
+
+.. bb:reporter:: Pushover
+
+.. index:: Pushover
+
+Pushover Notifications
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. py:class:: buildbot.reporters.pushover.PushoverNotifier
+
+Apart of sending mail, Buildbot can send Pushover_ notifications. It can be used by administrators to receive an instant message to an iPhone or an Android device if a build fails. The :class:`PushoverNotifier` reporter is used to accomplish this. Its configuration is very similar to the mail notifications, however—due to the notification size constrains—the logs and patches cannot be attached.
+
+To use this reporter, you need to generate and application on the Pushover website https://pushover.net/apps/ and provide your user key and API token.
+
+The following simple example will send an email upon the completion of each build.
+The email contains a description of the :class:`Build`, its results, and URLs where more information can be obtained. The ``user_key`` and ``api_token`` values should be replaces with proper ones obtained from the Pushover website for your application.
+
+::
+
+    from buildbot.plugins import reporters
+    pn = reporters.PushoverNotifier(user_key="1234", api_token='abcd')
+    c['services'].append(pn)
+
+
+This notifier supports parameters ``subject``, ``mode``, ``builders``, ``tags``, ``schedulers``, ``branches``, ``buildSetSummary``, ``messageFormatter``, ``watchedWorkers``, and ``messageFormatterMissingWorker`` from the mail notifier. See above for their explanation.
+However, ``watchedWorkers`` defaults to *None*.
+
+The following additional parameters are accepted by this class:
+
+``user_key``
+    The user key from the Pushover website. It is used to identify the notification recipient.
+
+``api_token``
+    API token for a custom application from the Pushover website.
+
```

Comment:
a link to api_token generation will be useful for people. Or we will get support requests from people putting their password in there